src/alph_puzzle.c

```diff
@@ -247,62 +646,60 @@ static int AlphPuzzleMainSeq_WaitForInput(AlphPuzzleData *data) {
     return AlphPuzzle_CheckInput(data);
 }
 
-static int AlphPuzzleMainSeq_2(AlphPuzzleData *data) {
-    return ov110_021E5CCC(data);
+static int AlphPuzzleMainSeq_PickupTile(AlphPuzzleData *data) {
+    return AlphPuzzleMainSeq_PickupTile_impl(data);
 }
 
-static int AlphPuzzleMainSeq_3(AlphPuzzleData *data) {
-    return ov110_021E5E1C(data);
+static int AlphPuzzleMainSeq_HoldingTile(AlphPuzzleData *data) {
+    return AlphPuzzleMainSeq_HoldingTile_impl(data);
 }
 
 static int AlphPuzzleMainSeq_RotateTile(AlphPuzzleData *data) {
-    return ov110_021E5F84(data);
+    return AlphPuzzleMainSeq_RotateTile_impl(data);
 }
 
-static int AlphPuzzleMainSeq_5(AlphPuzzleData *data) {
-    return ov110_021E6014(data);
+static int AlphPuzzleMainSeq_Quit(AlphPuzzleData *data) {
+    return AlphPuzzleMainSeq_Quit_impl(data);
 }
 
 static int AlphPuzzleMainSeq_Clear(AlphPuzzleData *data) {
-    return ov110_021E6070(data);
+    return AlphPuzzleMainSeq_Clear_impl(data);
 }
 
-static void ov110_021E5BE4(AlphPuzzleData *data) {
+static void AlphPuzzle_SetupGraphics(AlphPuzzleData *data) {
     AlphPuzzle_InitTileData(data);
-    ov110_021E61D0(data);
-    ov110_021E6394(data);
+    AlphPuzzle_AllocBackgroundBuffers(data);
+    AlphPuzzle_LoadBackgroundGraphics(data);
     AlphPuzzle_InitText(data);
-    ov110_021E6618(data);
-    ov110_021E6730(data);
-    Main_SetVBlankIntrCB(ov110_021E6110, data);
+    AlphPuzzle_CreateWindows(data);
+    AlphPuzzle_InitSpriteGraphics(data);
+    Main_SetVBlankIntrCB(AlphPuzzle_VBlankCB, data);
 }
 
-static void ov110_021E5C18(AlphPuzzleData *data) {
-    ov110_021E6748(data);
-    ov110_021E6650(data);
+static void AlphPuzzle_TeardownGraphics(AlphPuzzleData *data) {
```

Comment:
what does teardown refer to here? is this basically a graphics destructor?

Reply:
There's only so many synonyms for "delete/destroy/deinit/teardown"...